package.json

```diff
@@ -43,6 +43,11 @@
     "inline-styles",
     "react"
   ],
+  "files": [
+    "package.json",
+    "lib",
```

Comment:
I forked the project, installed it through node, but there wasn't any lib folder. The lib folder is build from the src folder, since that folder is not included, it cant build the lib folder. Am I missing something?